site/pages/review-disruption.page.tsx

```diff
@@ -0,0 +1,402 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { startCase } from "lodash";
+import { NextPageContext } from "next";
+import Link from "next/link";
+import { parseCookies } from "nookies";
+import { ReactElement } from "react";
+import Table from "../components/form/Table";
+import { BaseLayout } from "../components/layout/Layout";
+import { ConsequenceType, TransportMode } from "../constants/enum";
+import { Consequence, Disruption, SocialMediaPost } from "../interfaces";
+import { convertDateTimeToFormat, formatTime, splitCamelCaseToString } from "../utils";
+
+const title = "Review Disruption";
+const description = "Review Disruption page for the Create Transport Disruptions Service";
+
+interface ReviewDisruptionProps {
+    previousDisruptionInformation: Disruption;
+    previousConsequencesInformation: Consequence[];
+    previousSocialMediaPosts: SocialMediaPost[];
+}
+
+const createChangeLink = (key: string, href: string) => (
+    <Link key={key} className="govuk-link" href={href}>
+        Change
+    </Link>
+);
+
+const isOperatorOrNetworkUrl = (type: string) =>
+    `/create-consequence-${type === "Operator wide" ? "operator" : "network"}`;
```

Comment:
The `ConsequenceType` enum value could be used `/create-consequence-${type === ConsequenceType.operatorWide ? "operator" : "network"}`

Reply:
done